### PR TITLE
Resource mapping tests

### PR DESCRIPTION
**Pull request by jakub-dzon**
```release-note
NONE
```

Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: tests/basic_vm_import_test.go [hunk 1]
```diff
@@ -14,23 +14,20 @@ import (
...
-	storageClass = "local"
-)
-
 type basicVmImportTest struct {
 	framework *fwk.Framework
 }
 
 var _ = Describe("Basic VM import ", func() {
+	var (
+		vmID = "123"
```
Comment (pkliczewski) [inline tests/basic_vm_import_test.go:23]: What do you think about moving those constants somewhere from where we can reference them in many tests?
We could do it in different PR.

Comment (jakub-dzon) [reply tests/basic_vm_import_test.go:23]: :+1: I thought about it as well, will do it in a separate PR.

**Review by pkliczewski — APPROVED** [review]

**Review by masayag — APPROVED** [review]
